### PR TITLE
III-4700 Change Publish command permission to aanbodBewerken

### PR DESCRIPTION
**Pull request by LucWollants**
### Fixed
- Changed `Publish` command permission to `aanbodBewerken`

---
Ticket: https://jira.uitdatabank.be/browse/III-4700

### CODE REVIEW

File: tests/Event/Commands/Moderation/PublishTest.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace CultuurNet\UDB3\Event\Commands\Moderation;
+
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+use PHPUnit\Framework\TestCase;
+
+final class PublishTest extends TestCase
+{
+    /**
+     * @test
```
Comment (bertramakers) [inline tests/Event/Commands/Moderation/PublishTest.php:13]: Maybe reference that this is a `@bugfix` and add a `@see` link to the ticket so we don't forget why this was overridden later.

Comment (bertramakers) [inline tests/Event/Commands/Moderation/PublishTest.php:13]: Maybe reference that this is a `@bugfix` and add a `@see` link to the ticket so we don't forget why this was overridden later.

Comment (LucWollants) [reply tests/Event/Commands/Moderation/PublishTest.php:13]: Screenshot 2022-05-17 at 15 36 23

Comment (bertramakers) [reply tests/Event/Commands/Moderation/PublishTest.php:13]: Sure! 👍

**Pull request merged**